include/integratorxx/atomic_densities/SlaterAtomicShell.hpp

```diff
@@ -0,0 +1,102 @@
+#pragma once
+#include <vector>
+#include <cmath>
+
+class SlaterTypeAtomicShell {
+  /// Angular momentum
+  int angular_momentum;
+  /// Exponents
+  std::vector<double> exponents;
+  /// Principal quantum numbers
+  std::vector<int> quantum_numbers;
+  /// Contraction orbital_coefficients
+  std::vector<std::vector<double>> orbital_coefficients;
+  /// Alpha orbital occupations
+  std::vector<int> alpha_occupations;
+  /// Beta orbital occupations
+  std::vector<int> beta_occupations;
+ public:
+  /// Dummy constructor
+  SlaterTypeAtomicShell() {};
+  /// Constructor
+  SlaterTypeAtomicShell(int angular_momentum_, std::vector<double> & exponents_, std::vector<int> & quantum_numbers_, std::vector<std::vector<double>> & coefficients_, std::vector<int> & alpha_occupations_, std::vector<int> & beta_occupations_) : angular_momentum(angular_momentum_), exponents(exponents_), quantum_numbers(quantum_numbers_), orbital_coefficients(coefficients_), alpha_occupations(alpha_occupations_), beta_occupations(beta_occupations_) {};
```

Comment:
If you make the arguments by const-reference or by value, the code should compile